src/main/scala/nl.knaw.dans.easy.deposit/docs/UserInfo.scala

```diff
@@ -25,24 +26,35 @@ case class UserInfo(userName: String,
                     prefix: Option[String] = None,
                     lastName: String,
                     displayName: String,
+                    email: String,
                    )
-object UserInfo {
+object UserInfo extends DebugEnhancedLogging {
   def apply(input: JsonInput): Try[UserInfo] = input.deserialize[UserInfo]
 
   def apply(attributes: Map[String, Seq[String]]): UserInfo = {
-    // For possible attribute keys see: https://github.com/DANS-KNAW/dans.easy-test-users/blob/master/templates
+
+    def getAttribute(key: String) = {
+      attributes
+        .getOrElse(key, Seq.empty)
+        .headOption
+        .getOrElse {
+          logger.warn(s"user has no attribute '$key' $attributes")
+          ""
+        }
+    }
     new UserInfo(
 
       // mandatory: https://github.com/DANS-KNAW/dans.easy-ldap-dir/blob/f17c391/files/easy-schema.ldif#L83-L84
-      userName = attributes.getOrElse("uid", Seq.empty).headOption.getOrElse(""),
+      userName = getAttribute("uid"),
 
       firstName = attributes.getOrElse("cn", Seq.empty).headOption,
```

Comment:
why not let this use `getAttribute` as well? In other words, why is `firstName` optional?